input-output-hk/lace
# feat(staking,common,extension): add details drawer in staking

# Checklist

- [x] JIRA - LW-6678

---

## Proposed solution

This PR reintroduces the pool details drawer in the staking package and necessary refactoring for common components e.g. Banner.

## Testing

1. Open Staking
2. Go to the **Browse Pools** tab
3. Click on a Pool
4. Observe the Stake Pool Details Drawer being displayed

## Screenshots
<img width="1512" alt="image" src="https://github.com/input-output-hk/lace/assets/17825044/988571e1-2a73-4f5e-a69f-2fa639d6f280">


---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@d1terqjryk7mu9.cloudfront.net/linux/chrome/684/5409477544/index.html) for [8f0b9557](https://github.com/input-output-hk/lace/pull/193/commits/8f0b95579f4b9c9625224e726f898cf6ef2fcaa5)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 34     | 2      | 0       | 0     | 36    | ❌     |
